### ISSUE TITLE
Help Deploy our Python Website with Bard API on PythonAnywhere

### ISSUE THREAD

**Issue by MohamedMafaz** [MohamedMafaz/Propoplex#11]
Hi there, amazing developers! 👋

We have an exciting project in the works—an exquisite website built using Python's index.py file, which generates stunning proposals with the help of the Bard API. However, we've hit a roadblock with GitHub Pages, as it always redirects to the index.html file instead. To overcome this hurdle and ensure seamless deployment, we're on the lookout for a talented contributor to help us deploy our website on PythonAnywhere.

PythonAnywhere is a fantastic platform that allows us to host and run Python applications effortlessly. By leveraging its capabilities, we can unleash the full potential of our proposal generation website, making it available to the world with ease.

If you're passionate about web development, Python, and deploying awesome projects, we would love to have you on board as a contributor! By participating in this endeavor, you'll gain hands-on experience with Python web applications, APIs, and deployment processes. Plus, you'll be making a meaningful contribution to our project, helping us share the beauty of our proposals with a wider audience.

To get started, all you need to do is provide us with your email ID, and we'll grant you access to the source code. You'll have the opportunity to dive into the codebase, understand its intricacies, and collaborate with our team to deploy the website successfully on PythonAnywhere.

So, if you're up for the challenge and eager to join an exciting project, please drop your email ID in the comments below. We'll reach out to you shortly with further instructions and provide any assistance you may need along the way.

Let's embark on this journey together and create an awe-inspiring platform that showcases the power of Python and the elegance of our proposal generation website. We can't wait to have you on board!

Thank you,
Mohamed Mafaz,
Propoplex

**Comment by therealtmmy** [comment]
Alright. Let me give this a try.
oluwatimilehinsamson@gmail.com is my email address

**Comment by MohamedMafaz** [comment]
Hey @therealtmmy 
I sent you a email check it out. If you do with any new email ID for propoplex. That'll be awesome. So I can get access for it

**Comment by MohamedMafaz** [comment]
Hey @therealtmmy 
Any Updates?

**Comment by therealtmmy** [comment]
Have you tried developing it one Vercel aside using github pages?

**Comment by MohamedMafaz** [comment]
Yeah! I tried Vercel, but they only provide hosting for static website. If you need I'll send the username and password in your email.

**Comment by MohamedMafaz** [comment]
Hey @therealtmmy 
Any Updates? May I know when it will be completed?

**Comment by therealtmmy** [comment]
At all. I know know of Vercel to be the best place to host it. You might want to try out paid hosting platform. Check out [www.whogohost](https://whogohost.com/)

**Comment by MohamedMafaz** [comment]
What about PythonAnywhere?

**Comment by MohamedMafaz** [comment]
Waiting for devender 15
